CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 **BUG FIXES**
 - Do not configure GPUs in Slurm when NVIDIA driver is not installed.
 - Fix the way `ExtraChefAttributes` are merged into the final configuration.
+- Fix sporadic failure of build-image due to missing moreutils
```

Comment:
I think we can stay generic and let `moreutils`be an example, something like

`Fix sporadic failure of build image process due to missing package, e.g. `No candidate version available for moreutils`

Reply:
Addressed to be more general.